content/academy/data_collection_basics/js_in_html.md

```diff
@@ -0,0 +1,51 @@
+---
+title: JavaScript in HTML
```

Comment:
Maybe "JavaScript data in HTML" ? This title seems like the text is gonna be about adding scripts inside HTML.